atlasdb-ete-tests/build.gradle

```diff
@@ -52,6 +52,10 @@ task longTest(type: Test) {
 task timeLockTest(type: Test) {
     dependsOn prepareForEteTests, ':timelock-server-distribution:dockerTag'
     include '**/*TimeLock*.class'
+
+    // TODO (jkong): Re-enable migration tests once a new story for handling volumes-in-circle-2 is available.
```

Comment:
We use the machine executor internally to do dockerized testing with volumes

Reply:
Yep, I've seen this in $INTERNAL_SECURITY_PRODUCT as well. Thanks